MarkBind/markbind
# [DO NOT MERGE] Move some script elements (demo)

These `script` elements found in `Page.njk` have been moved down to be just above the `body` closing tag:

```
<script src="{{ asset.vue }}"></script>
<script src="{{ asset.markBindJs }}"></script>
<script src="{{ asset.pageVueRenderJs }}"></script>
<script src="{{ asset.jQuery }}"></script>
```

There are two commits: one for where the script elements are down below, and one for where they're up above. You can check-out between these commits to see the changes without manually modifying the code.

**Testing instructions:**

0. (optional) Create a new MarkBind project.
1. Check-out the commit that you want to test.
2. Serve the MarkBind project.
* Note: the following steps assume the use of Edge/Chrome; the same (or similar) instructions might be applicable to other browsers.
3. Go to the MarkBind serving address (if it isn't open already).
4. Right-click anywhere on the page and select _Inspect_.
5. Click on the _Network_ tab, then turn on _Disabling Cache_ and set the throttling option to _Slow 3G_.
6. Refresh the page; once the page is fully-loaded, a timestamp with the label _Load_ will appear on the bottom of _Network_ tab.
    * If the scripts are at the bottom during this step, then the [header padding FOUC](https://github.com/MarkBind/markbind/issues/1560) should take more time to happen.
7. Repeat Step 6 if more tests are required for the current commit.
8. Once finished, stop the MarkBind serving process.
9. Repeat Steps 1-8 for the other commit if necessary.
